core/include/log.h

```diff
@@ -73,7 +73,15 @@ enum {
 /**
  * @brief Log message if level <= LOG_LEVEL
  */
+#ifdef __clang__
+#define LOG(level, ...) \
```

Comment:
Please add a short comment here why we need to ignore this warning, and in what version of Clang you are seeing this problem. Just for future reference when someone finds this code and wonder why the special handling of Clang.